### PR TITLE
Feature/26-publicar-incidencias

### PR DESCRIPTION
**Pull request by felconmar**

### CODE REVIEW

File: src/Incidencia/ValidateIncidenciaForm.js [hunk 1]
```diff
@@ -0,0 +1,22 @@
+export default function ValidateIncidenciaForm(form){
+
+    let errors = {}
+   
+    // Validación título
+    if(!form.titulo.trim()){
+        errors.titulo= "El título es un campo obligatorio"
```
Comment (luibelzan) [inline src/Incidencia/ValidateIncidenciaForm.js:7]: Codacy has a fix for the issue: [Missing semicolon.](https://app.codacy.com/gh/park-inn-ispp/park-inn-frontend/pullRequest?prid=9179155)
```suggestion
        errors.titulo= "El título es un campo obligatorio"; 
```

File: src/Incidencia/ValidateIncidenciaForm.js [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+    // Validación título
+    if(!form.titulo.trim()){
+        errors.titulo= "El título es un campo obligatorio" 
+    } else if (form.titulo.length > 100 ){
+        errors.titulo= "El título no puede contener más de 100 caracteres"
+    }
+
+    // Validación descripcion
+    if(!form.descripcion.trim()){
+        errors.descripcion= "La descripción es un campo obligatorio"
```
Comment (luibelzan) [inline src/Incidencia/ValidateIncidenciaForm.js:14]: Codacy has a fix for the issue: [Missing semicolon.](https://app.codacy.com/gh/park-inn-ispp/park-inn-frontend/pullRequest?prid=9179155)
```suggestion
        errors.descripcion= "La descripción es un campo obligatorio"; 
```

File: src/Incidencia/ValidateIncidenciaForm.js [hunk 3]
```diff
@@ -0,0 +1,16 @@
+export default function ValidateIncidenciaForm(form){
+    let errors = {};
+    // Validación título
+    if(!form.titulo.trim())
+        errors.titulo= "El título es un campo obligatorio";
+    if (form.titulo.length > 100 )
+        errors.titulo= "El título no puede contener más de 100 caracteres";
+    // Validación descripcion
+    if(!form.descripcion.trim())
+        errors.descripcion= "La descripción es un campo obligatorio";
```
Comment (luibelzan) [inline src/Incidencia/ValidateIncidenciaForm.js:10]: Codacy has a fix for the issue: [Expected { after 'if' condition.](https://app.codacy.com/gh/park-inn-ispp/park-inn-frontend/pullRequest?prid=9179155)
```suggestion
        {errors.descripcion= "La descripción es un campo obligatorio";}
```

File: src/Incidencia/ValidateIncidenciaForm.js [hunk 4]
```diff
@@ -0,0 +1,16 @@
...
+    if(form.descripcion.length < 100)
+        errors.descripcion = "La decripción no puede contener menos de 100 caracteres";
```
Comment (luibelzan) [inline src/Incidencia/ValidateIncidenciaForm.js:12]: Codacy has a fix for the issue: [Expected { after 'if' condition.](https://app.codacy.com/gh/park-inn-ispp/park-inn-frontend/pullRequest?prid=9179155)
```suggestion
        {errors.descripcion = "La decripción no puede contener menos de 100 caracteres";}
```

File: src/Incidencia/ValidateIncidenciaForm.js [hunk 5]
```diff
@@ -0,0 +1,16 @@
...
+    if (form.descripcion.length > 500)
+        errors.descripcion = "La decripción no puede contener más de 500 caracteres";
```
Comment (luibelzan) [inline src/Incidencia/ValidateIncidenciaForm.js:14]: Codacy has a fix for the issue: [Expected { after 'if' condition.](https://app.codacy.com/gh/park-inn-ispp/park-inn-frontend/pullRequest?prid=9179155)
```suggestion
        {errors.descripcion = "La decripción no puede contener más de 500 caracteres";}
```

File: src/Incidencia/ValidateIncidenciaForm.js [hunk 6]
```diff
@@ -0,0 +1,16 @@
+export default function ValidateIncidenciaForm(form){
+    let errors = {};
+    // Validación título
+    if(!form.titulo.trim())
+        errors.titulo= "El título es un campo obligatorio";
+    if (form.titulo.length > 100 )
+        errors.titulo= "El título no puede contener más de 100 caracteres";
```
Comment (luibelzan) [inline src/Incidencia/ValidateIncidenciaForm.js:7]: Codacy has a fix for the issue: [Expected { after 'if' condition.](https://app.codacy.com/gh/park-inn-ispp/park-inn-frontend/pullRequest?prid=9179155)
```suggestion
        {errors.titulo= "El título no puede contener más de 100 caracteres";}
```

File: src/Incidencia/ValidateIncidenciaForm.js [hunk 7]
```diff
@@ -0,0 +1,16 @@
+export default function ValidateIncidenciaForm(form){
+    let errors = {};
+    // Validación título
+    if(!form.titulo.trim())
+        errors.titulo= "El título es un campo obligatorio";
```
Comment (luibelzan) [inline src/Incidencia/ValidateIncidenciaForm.js:5]: Codacy has a fix for the issue: [Expected { after 'if' condition.](https://app.codacy.com/gh/park-inn-ispp/park-inn-frontend/pullRequest?prid=9179155)
```suggestion
        {errors.titulo= "El título es un campo obligatorio";}
```

**Review by vicdiacor — APPROVED** [review]

**Pull request merged**